app/Services/ExportService.php

```diff
@@ -58,6 +65,6 @@ public function exportSignups($campaignId)
             $writer->insertOne($nextRow);
         }
 
-        return $writer->output('export_' . $campaignId . '.csv');
+        return $writer->__toString();
```

Comment:
Let's return `(string) $writer` rather than explicitly calling the magic method! 🎩